### PR TITLE
Add JakobDev.jdTextEdit

### PR DESCRIPTION
**Pull request by JakobDev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94297)

### CODE REVIEW

**Comment by stephengillie** [comment]
I believe the issue is heavy bandwidth throttling, possibly based on user agent. My sandbox sees a very slow download, and got stuck with an incomplete download at 28 MB of 43 MB. Meanwhile a browser on the same PC gets the package in a few seconds.

File: manifests/j/JakobDev/jdTextEdit/10.3/JakobDev.jdTextEdit.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+PackageIdentifier: JakobDev.jdTextEdit
+PackageVersion: 10.3
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Installers:
```
Comment (stephengillie) [inline manifests/j/JakobDev/jdTextEdit/10.3/JakobDev.jdTextEdit.installer.yaml:5]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Comment (stephengillie) [reply manifests/j/JakobDev/jdTextEdit/10.3/JakobDev.jdTextEdit.installer.yaml:5]: The package installs normally, but needs the dependency to run.
image

Comment (JakobDev) [reply manifests/j/JakobDev/jdTextEdit/10.3/JakobDev.jdTextEdit.installer.yaml:5]: The Bot says, that dependencies are currently not supported. Should I just upload a new Installer that has everything included?

Comment (stephengillie) [reply manifests/j/JakobDev/jdTextEdit/10.3/JakobDev.jdTextEdit.installer.yaml:5]: Yes, bundling dependencies into your installer is currently the recommended method. Dependency support is becoming more robust as newer versions are released, but might still be experimental in some situations.

Comment (JakobDev) [reply manifests/j/JakobDev/jdTextEdit/10.3/JakobDev.jdTextEdit.installer.yaml:5]: I have now updated the Installer. It should now work without dependencies.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**